### PR TITLE
Add a Brave browser class and function

### PR DESCRIPTION
**Pull request by bjlange**
Can confirm this works on my macOS machine, but I don't have an easy way to test on Windows or Linux at the moment... I looked up the file locations in the Brave documentation so they should be correct, but haven't been able to test myself.

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -424,6 +424,27 @@ def __init__(self, cookie_file=None, domain_name="", key_file=None):
...
+                    {'env':'LOCALAPPDATA', 'path':'BraveSoftware\\Brave-Browser\\User Data\\Default\\Cookies'},
+                    {'env':'APPDATA', 'path':'BraveSoftware\\Brave-Browser\\User Data\\Default\\Cookies'}
+            ],
+            'osx_cookies': ['~/Library/Application Support/BraveSoftware/Brave-Browser/Default/Cookies'],
+            'windows_keys': [
+                    {'env':'APPDATA', 'path':'..\\Local\\BraveSoftware\\Brave-Browser\\User Data\\Local State'},
+                    {'env':'LOCALAPPDATA', 'path':'BraveSoftware\\Brave-Browser\\User Data\\Local State'},
+                    {'env':'APPDATA', 'path':'BraveSoftware\\Brave-Browser\\User Data\\Local State'}
+            ],
+            'os_crypt_name':'chromium',
```
Comment (rafiibrahim8) [inline __init__.py:442]: @bjlange os_crypt_name for Brave Browser is `brave`. Other than that everything works perfectly. Tested on Linux and Windows.

**Comment by rafiibrahim8** [comment]
@bjlange Can you add dev and beta channels too?

**Comment by drmikecrowe** [comment]
@rafiibrahim8 @bjlange -- what else is required to get this merged into main?
@rafiibrahim8 -- Can you please explain the dev/beta channels request?  I'll help but I don't understand that task

**Comment by rafiibrahim8** [comment]
@drmikecrowe 

> @rafiibrahim8 @bjlange -- what else is required to get this merged into main?

[This](https://github.com/borisbabic/browser_cookie3/pull/73#discussion_r579361779) needs to be fixed.

> @rafiibrahim8 -- Can you please explain the dev/beta channels request? I'll help but I don't understand that task

Brave browser gets released in 3 channels: stable, beta, dev. Each channel has a  different directory where it saves the cookies.

For example, here is the implementation for chrome. It checks cookies for stable and beta channels.
https://github.com/borisbabic/browser_cookie3/blob/303d5bfc499bae94f2a4455ef60f89bfca474b75/__init__.py#L363-L366

**Comment by drmikecrowe** [comment]
@rafiibrahim8 I created a branch with these changes and tested.  Unfortunately, in ubuntu 20.10, Brave Version 1.26.74 Chromium: 91.0.4472.124 (Official Build) (64-bit) I get:

```
  File "/home/mcrowe/.pyenv/versions/3.8.7/lib/python3.8/site-packages/browser_cookie3/__init__.py", line 638, in brave
    return Brave(cookie_file, domain_name, key_file).load()
  File "/home/mcrowe/.pyenv/versions/3.8.7/lib/python3.8/site-packages/browser_cookie3/__init__.py", line 452, in __init__
    super().__init__(browser='Brave', cookie_file=cookie_file, domain_name=domain_name, key_file=key_file, **args)
  File "/home/mcrowe/.pyenv/versions/3.8.7/lib/python3.8/site-packages/browser_cookie3/__init__.py", line 192, in __init__
    self.__add_key_and_cookie_file(**kwargs)
  File "/home/mcrowe/.pyenv/versions/3.8.7/lib/python3.8/site-packages/browser_cookie3/__init__.py", line 214, in __add_key_and_cookie_file
    my_pass = get_linux_pass(os_crypt_name)
  File "/home/mcrowe/.pyenv/versions/3.8.7/lib/python3.8/site-packages/browser_cookie3/__init__.py", line 141, in get_linux_pass
    my_pass = keyring.get_password(
AttributeError: 'NoneType' object has no attribute 'encode'
```

Can you point me to additional diagnostics to resolve?

I checked the `secretstorage` items, and there are only 8 or so items in that collection and the only browser is `Chrome Safe Storage Control` -- nothing brave related

**Comment by borisbabic** [comment]
Added in #91

**Pull request closed without merging**